subsys/net/ip/tcp2.c

```diff
@@ -1435,17 +1435,25 @@ struct net_tcp_hdr *net_tcp_input(struct net_pkt *pkt,
 static sys_slist_t tp_q = SYS_SLIST_STATIC_INIT(&tp_q);
```

Comment:
The commit subject and body has same information in commit "net: tcp2: Support arbitrary requests in tcp_win_pop()" and couple of others too. I would suggest that this is not done like this, the subject in this case is fine, but the body of the commit message should answer the question "why this change is done" and it is not doing it now.